inputsources/composite/stackoutput.ftl

```diff
@@ -0,0 +1,43 @@
+[#ftl]
+
+[#-- Get stack output --]
+[#function azure_input_composite_stackoutput_filter outputFilter]
+  [#return 
+    {
+      "Subscription" : (outputFilter.Account)!accountObject.AWSId,
+      "Region" : outputFilter.Region,
+      "DeploymentUnit" : outputFilter.DeploymentUnit
+    }
+  ]
+[/#function]
+
+[#macro azure_input_composite_stackoutput_seed]
+
+  [#local deploymentOutputs = []]
+
+  [#-- ARM Stack Output Processing --]
+  [#list commandLineOptions.Composites.StackOutputs as deploymentTemplate]
+
+    [#local level = ((deploymentTemplate["FileName"])?split('-'))[0]]
+    [#list (deploymentTemplate["Content"]![]) as rawDeploymentOutput]
+      [#if (rawDeploymentOutput["properties"]["outputs"]!{})?has_content]
+
+        [#local deploymentOutput = {
```

Comment:
I would still call these stack outputs. Stackoutputs are the codeontap name for them rather than the provider name